### PR TITLE
Docs to match #32908 - REPLICA_DB_TASKS / REPLICA_PREPARED_STATEMENTS

### PR DESCRIPTION
**Pull request by shleeable**
https://github.com/mastodon/mastodon/pull/32908

### CODE REVIEW

File: content/en/admin/scaling.md [hunk 1]
```diff
@@ -412,6 +412,8 @@ REPLICA_DB_PORT
 REPLICA_DB_NAME
 REPLICA_DB_USER
 REPLICA_DB_PASS
+REPLICA_PREPARED_STATEMENTS (defaults to true)
```
Comment (ClearlyClaire) [inline content/en/admin/scaling.md:415]: In the related PR, it actually defaults to the same value as `PREPARED_STATEMENTS`, not to `true`: if `PREPARED_STATEMTNTS` is set to anything but `true`, so will `REPLICA_PREPARED_STATEMENTS` by default.

File: content/en/admin/scaling.md [hunk 2]
```diff
@@ -412,6 +412,8 @@ REPLICA_DB_PORT
...
+REPLICA_DB_TASKS (defaults to true)
```
Comment (ClearlyClaire) [inline content/en/admin/scaling.md:416]: I think this warrants some explanation.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

File: content/en/admin/scaling.md (hunk 2 above)
Comment (shleeable) [reply content/en/admin/scaling.md]: Expanded documentation.

File: content/en/admin/scaling.md (hunk 1 above)
Comment (shleeable) [reply content/en/admin/scaling.md]: Expanded documentation. Please review.

**Comment by shleeable** [comment]
@ClearlyClaire I hope this is more suitable.

File: content/en/admin/scaling.md [hunk 3]
```diff
@@ -412,10 +412,16 @@ REPLICA_DB_PORT
 REPLICA_DB_NAME
 REPLICA_DB_USER
 REPLICA_DB_PASS
+REPLICA_PREPARED_STATEMENTS
+REPLICA_DB_TASKS
 ```
 
 Alternatively, you can also use `REPLICA_DATABASE_URL` if you want to configure them all using the same variable.
 
+`REPLICA_DB_TASKS` will connect to an replica database without any database mangement tasks such as schema management, migrations, seeds, etc. By default it is set to true.
```
Comment (ClearlyClaire) [inline content/en/admin/scaling.md:421]: Isn't this the opposite? Reading this comment, I'm under the impression that `REPLICA_DB_TASKS=true` means the database management tasks don't apply to the replica.

Comment (shleeable) [reply content/en/admin/scaling.md]: I've added `=false` to make the statement less confusing.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**